build/kubernetes/coredns_deployment_patch.yaml

```diff
@@ -0,0 +1,15 @@
+spec:
+  template:
+    spec:
+      containers:
+      - name: coredns
+        image: localhost:5000/coredns
+      volumes:
+      - name: config-volume
+        configMap:
+          name: coredns
+          items:
+          - key: Corefile
```

Comment:
i think we can remove this key

Reply:
If I keep the `Zonefile` Key and path only, the deployment is patched by updating `Corefile` with `Zonefile`. I need to keep both to be able to update the deployment with both values.